JavaScript/JavaScriptSDK/Context/Session.ts

```diff
@@ -71,7 +71,7 @@ module Microsoft.ApplicationInsights.Context {
                 this.initializeAutomaticSession();
             }
 
-            var now = +new Date;
+            var now = dateTime.Now();
```

Comment:
We use it to set a timestamp of each event and sent it to AI backend, so it has to be a Date. The code will fall back to `Date().getTime()` if `window.performance` is not available.